assets/src/scripts/components/graph/view/lithology.js

```diff
@@ -0,0 +1,42 @@
+import { transition } from 'd3-transition';
+
+const LITHOLOGY_COLORS = [
+    '#9A887E',
+    '#72655C',
+    '#CDE569',
+    '#C6BBB7'
+];
+
+
+export default function (elem, {lithology}, container) {
+    container = container || elem
+        .append('g');
+            //.style('opacity', '.5');
```

Comment:
Is there a reason this is commented out?

Reply:
It doesn't look good - I'll remove